src/main/resources/bug_report.md

```diff
@@ -0,0 +1,41 @@
+## Bug description
+Please include steps to reproduce (like `go to...`/`click on...` etc.) + expected and actual behavior.
+
+For more non-trivial issues, we would also appreciate if you included the following details:
+
+### Running environment
+ - Git Machete plugin version - %macheteVersion%
+ - IDE %ide%
+ - Operating system [e.g. Windows]
```

Comment:
Is it possible to extract it from IntelliJ APIs? or maybe via some `Runtime#exec`

Reply:
Good question. Will check that.